packages/cli-test/src/cli/cli-process.ts

```diff
@@ -1,7 +1,6 @@
-import { shell } from './shell';
-
 import type { SpawnOptionsWithoutStdio } from 'node:child_process';
 import type { ShellProcess } from '../types/shell';
+import { shell } from './shell';
```

Comment:
note: biome requires the imports to be sorted alphabetically based on their `from`.